src/core/quicdef.h

```diff
@@ -495,6 +495,11 @@ CXPLAT_STATIC_ASSERT(
 //
 #define QUIC_DEFAULT_DEST_CID_UPDATE_IDLE_TIMEOUT_MS 20000
 
+//
+// The default value for enabling grease quic bit extension.
+//
+#define QUIC_DEFAULT_GREASE_QUIC_BIT_ENABLED         TRUE
```

Comment:
I want to (at least for now) disable this by default.

Reply:
Alright!